src/components/views/elements/Flair.js

```diff
@@ -117,7 +117,7 @@ export default class Flair extends React.Component {
 
     render() {
         if (this.state.profiles.length === 0) {
-            return <div />;
```

Comment:
Huh, surprised it used to switch to a `div` when empty... Anyway, now it's fixed!